code/dqe/schemacheck.q

```diff
@@ -0,0 +1,6 @@
+\d .dqe
+schemacheck:{[tab;schema;colname]
```

Comment:
I would say "schema" is the names, types and attributes rather than just types :eyes: my 2 cents

Reply:
There is an attribute check already, and this schemacheck function checks both col name and types. Would you say combining both attribute check and schemacheck makes sense?